packages/desktop-client/e2e/reports.test.js

```diff
@@ -34,4 +34,24 @@ test.describe('Reports', () => {
     expect(reports).toEqual(['Net Worth', 'Cash Flow']);
     await expect(page).toHaveScreenshot(screenshotConfig(page));
   });
+
+  test('loads net worth graph and checks visuals', async () => {
+    await reportsPage.goToNetWorthPage();
+
+    // Move cursor to the middle of the page to show tooltip
+    const viewportSize = await page.viewportSize();
+    await page.mouse.move(viewportSize.width / 2, viewportSize.height / 2);
+
+    await expect(page).toHaveScreenshot(screenshotConfig(page));
+  });
+
+  test('loads cash flow graph and checks visuals', async () => {
+    await reportsPage.goToCashFlowPage();
+
+    // Move cursor to the middle of the page to show tooltip
+    const viewportSize = await page.viewportSize();
+    await page.mouse.move(viewportSize.width / 2, viewportSize.height / 2);
+
+    await expect(page).toHaveScreenshot(screenshotConfig(page));
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would creating screenshots of only the popover make the tests more stable?

Reply:
I changed it to take screenshots of the overall page instead of trying to include the tooltips. I tried add test-ids to the tooltips and taking screenshots of those as well, but it was also flaky. I think this is good enough for now